suite-native/module-accounts-import/src/screens/ScanQRCodeModalScreen.tsx

```diff
@@ -8,20 +8,26 @@ import {
     ScreenHeader,
     StackProps,
 } from '@suite-native/navigation';
+import { isEthereumAddressLikeNetwork } from '@suite-common/wallet-config';
 
 export const ScanQRCodeModalScreen = ({
     navigation,
     route,
 }: StackProps<AccountsImportStackParamList, AccountsImportStackRoutes.XpubScanModal>) => {
+    const { networkSymbol } = route.params;
     const handleBarCodeScanned = (data: string) => {
         navigation.navigate(AccountsImportStackRoutes.XpubScan, {
             qrCode: data,
-            networkSymbol: route.params.networkSymbol,
+            networkSymbol,
         });
     };
 
+    const screenTitle = isEthereumAddressLikeNetwork(networkSymbol)
+        ? 'Scan receive address'
+        : 'Scan public key (XPUB)';
```

Comment:
duplication with xpubimportsection.tsx. Create a function that returns string and reuse it

Reply:
Does it make sense to create a function for this? Where should it be placed? IMHO function for such a simple operation which is used only in two places is not necessary. It would be easily forgotten as a dead code if the content of those two components would change in the future.  Of course, if it would be used all over the app, then I would agree 100%.  But if you insist no problem. But where should it be placed?

Reply:
depracated since 72f5f54b5